src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -909,7 +913,9 @@ export default async ({ req, res, log }) => {
         case 'json':
             return res.json({type: "This is a JSON response"});
         case 'binary':
-            return res.binary(InputFile.fromPath('/path/to/file.png', 'file.png'));
+            const fileContents = Deno.readFileSync('/path/to/file.png');
+            // or const fileContents = await Deno.readFile('/path/to/file.png');
```

Comment:
everywhere you gave 2 options, lets always go with the better one. that's always async version